hello-word/src/App.js

```diff
@@ -1,9 +1,10 @@
-import Button from "./components/Button"
 
 export default function App() {
-    return (
-       <div>
-        <Button />
-       </div>
-    )
-}
\ No newline at end of file
+  const itemList = ['dog', 'cat', 'chicken', 'cow', 'sheep', 'horse'].map(item => <li>{item}</li>)
```

Comment:
Don't forget to add a 'key' when mapping elements in React. It helps React work more efficiently.